objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/LocationMapper.java

```diff
@@ -97,12 +100,35 @@
                 if (o instanceof Number) {
                     xr = ((Number) o).floatValue();
                 }
+
                 if (yt == null || yb == null || xr == null || xl == null) {
                     throw new IllegalArgumentException(this.getClass().getName() +
                             ".missingcoords: Input missing coordinates.");
                 }
+
+                o = loc.get("centerY");
```

Comment:
We try & be backwards compatible -- if we don't get a center, build one.  This is more an issue in the VA server, but we handle it here as well.  This may help for other connectors originally built from this one.